build.gradle

```diff
@@ -14,103 +14,56 @@
  * limitations under the License.
  */
 plugins {
-    id 'java'
-    id 'maven'
+    id 'java-library'
+    id 'maven-publish'
     id 'signing'
-    id 'eclipse' // Only used so the Eclipse STS Gradle plugin can see the 'perf' source set dependencies. :-(
     id 'checkstyle'
     id 'idea'
 
     id 'org.asciidoctor.jvm.convert' version '3.1.0'
 }
 
-asciidoctor {
-    languages 'en'
-    resources {
-        from(javadoc) {
-            into './javadoc'
-        }
-        from('src/docs/resources') {
-            into './resources'
-        }
-        from('src/docs/files') {
-            into './files'
-        }
-    }
-    attributes 'source-highlighter': 'coderay',
-            'toc': 'left',
-            'xrefstyle': 'short'
-    copyResourcesOnlyIf 'html5'
-    baseDirFollowsSourceFile()
-}
+group = 'com.lmax'
+version = new Version(major: 4, minor: 0, revision: 0, snapshot: true)
```

Comment:
You changed the version here, but did not update `CHANGELOG.md`. I suppose you'd want to.

Reply:
Disregard. It already has been updated.